app/handlers/retrieveResults.py

```diff
@@ -16,5 +21,32 @@ class RetrieveResultsHandler(tornado.web.RequestHandler):
 
     @apiError error_message The error message to display.
     """
+
+    def post(self):
+        self.identifier = self.get_body_arguments("identifier")
+
     def get(self):
-        self.finish("Retrieve Results")
\ No newline at end of file
+        identifier = self.get_body_arguments("identifier")
+        project_path = get_project_path(identifier[0])
```

Comment:
why is this identifier[0]? this should be just identifier. If this is wrong, it is due to a bug in the GUI cloud_api code.